lib/queue.js

```diff
@@ -184,7 +191,10 @@ class Queue extends Emitter {
     const cleanup = () => {
       this._isClosed = true;
 
-      const clients = [this.client];
+      const clients = [];
+      if (this.settings.quitCommandClient) {
```

Comment:
We still want to use `quit`, as it tells Redis to disconnect, and ensures we flush our command queue in the process. We can't use `quit` with `bclient` because chances are it's waiting for a job, with no timeout.